ietf-wdm-tunnel.tree

```diff
@@ -29,26 +30,30 @@ module: ietf-wdm-tunnel
     |     +--rw otsi-carrier-id               uint16
     |     +--rw transceiver-id?               uint32
     |     +--rw operational-modes*            string
+    |     +--rw otsi-carrier-frequency?       l0-types:frequency-thz
     |     +--rw tx-tune-constraints
     |     |  +--rw min-central-frequency?    frequency-thz
     |     |  +--rw max-central-frequency?    frequency-thz
     |     |  +--rw transceiver-tunability?   frequency-ghz
     |     +--rw line-coding-bitrate*          identityref
     |     +--rw tx-channel-power?             l0-types:power-dbm
     |     +--rw preferred-rx-channel-power?   l0-types:power-dbm
+    |     +--rw gsnr-extra-margin?            snr
```

Comment:
Add YANG description to indicate that gsnr-extra-margin applies to an OMS section

Reply:
the grouping containing gsnr-extra-margin is defined in l0-types, where the description indicates that it applies to the OSNR-min of the transceiver (which terminates an OTSi):
"An additional margin to be added to the OSNR-min of the
        transceiver when checking the estimated received Generalized
        SNR (GSNR)."
Perhaps keep the description as is?